lib/src/main/java/graphql/nadel/engine/blueprint/NadelExecutionBlueprintFactory.kt

```diff
@@ -316,9 +316,7 @@ private class Factory(
             NadelBatchHydrationMatchStrategy.MatchIndex
         } else if (hydration.objectIdentifiers?.isNotEmpty() == true) {
             NadelBatchHydrationMatchStrategy.MatchObjectIdentifiers(
-                hydration.objectIdentifiers
-                // todo remove !! once they we merge api/ and nextgen-engine/
-                !!.map { objectIdentifier ->
+                hydration.objectIdentifiers.map { objectIdentifier ->
```

Comment:
Previously the Kotlin compiler couldn't _ensure_ that the value would be final as the code belonged to a different module e.g. module could be swapped out, different versions might not be final etc.